docs/detections/rules-ui-manage.asciidoc

```diff
@@ -16,7 +16,7 @@ On the Detection rules page, you can:
 === Load and activate prebuilt Elastic rules
 
 To load the {es-sec-app}'s <<prebuilt-rules, prebuilt rules>>, click
-*Load Elastic prebuilt rules* on the *Detection rules* page (*Detections* -> *Manage detection rules* ->
+*Load Elastic prebuilt rules* on the *Rules* page (*Detect* -> *Rules* ->
```

Comment:
FYI: I was not able to observe this behavior directly in the latest dev server, because it was already loaded with prebuilt rules. I updated this assuming that the only thing changed is the navigation to get to the Rules page, not the name of the buttons/links to load the prebuilt rules.